roles/slave_tools/tasks/nodejs.yml

```diff
@@ -169,11 +169,29 @@
     PATH: /opt/build/tools/node-v10.13.0-linux-x64/bin:{{ansible_env.PATH}}
   tags: nodejs
 
+- name: Check for Node 12.13 LTS
+  stat: path=/opt/build/tools/node-v12.13.0-linux-x64
+  register: nodelts_installed
+- name: Get Node 12.13 package
+  s3: bucket={{s3_bucket}} region={{s3_region}}
+      object=/{{s3_tools_path}}/node-v12.13.0-linux-x64.tar.xz dest=/tmp/node-v12.13.0-linux-x64.tar.xz mode=get
+      aws_access_key={{aws_id.msg}} aws_secret_key={{aws_secret.msg}}
+  when: not nodelts_installed.stat.exists
+- name: Install Node 12.13
+  unarchive: src=/tmp/node-v12.13.0-linux-x64.tar.xz dest=/opt/build/tools copy=no owner=root group=root
+  when: not nodelts_installed.stat.exists
+- name: Install npm 6.12.0
+  npm: name=npm version=6.12.0 global=yes
+  environment:
+    PATH: /opt/build/tools/node-v12.13.0-linux-x64/bin:{{ansible_env.PATH}}
+  tags: nodejs
```

Comment:
@nelsonsilva I've not added any dependencies because I don't need them and they should be local and can be run with `npx`